client/src/components/SalesPitch.jsx

```diff
@@ -2,25 +2,27 @@ import React from 'react';
 import VerticalRule from './VerticalRule.jsx';
 
 var SalesPitch = function(props) {
```

Comment:
I see some different approaches for using var/let/const for different components. I am practicing an approach of starting with const unless I know something is going to be mutable or need to be hoisted. As React stateless functional components are intended to be immutable, I keep them as const to help avoid any future bugs that may creep in.  It may be worth a quick convo to make a consistent decision w/ the team on this.

Reply:
Good point.